test-lib-openshift.sh

```diff
@@ -392,7 +399,13 @@ function ct_os_delete_project() {
 # Deletes all objects within the project.
 # Handy when we have one project and want to run more tests.
 function ct_delete_all_objects() {
-  for x in bc builds dc is isimage istag po pv pvc rc routes secrets svc ; do
+  objects="bc builds dc is isimage istag po pv pvc rc routes secrets svc"
```

Comment:
I would prefer to set `objects` as a local variable.

Reply:
﻿﻿﻿Fixed by https://github.com/sclorg/container-common-scripts/pull/280/commits/0498bcb4329668ccbb026795e99315932021ff83